sql-native/src/main/java/org/ballerinalang/sql/datasource/SQLDatasource.java

```diff
@@ -406,7 +406,12 @@ private AtomikosDataSourceBean buildXAAwareDataSource(SQLDatasourceParams sqlDat
     }
 
     private String buildErrorMessage (Throwable t) {
-        StringBuilder message = new StringBuilder("Error in SQL connector configuration: " + t.getMessage() + "");
+        String msg = t.getMessage();
+        if (msg.contains("ClassNotFoundException")) {
+            return "Couldn't find the appropriate driver. Check whether Ballerina supports or not the given driver " +
+                    "version and the driver path in the `Ballerina.toml` is correct or not.";
```

Comment:
```suggestion
            return "Error while loading database driver. This may be because the database driver path is not configured correctly in the `Ballerina.toml` file or provided database driver version is not supported by the connector";
```